src/schema/v2/artwork/index.ts

```diff
@@ -69,6 +69,33 @@ const has_multiple_editions = (edition_sets) => {
   return edition_sets && edition_sets.length > 1
 }
 
+const ArtworkConsignmentSubmissionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "ArtworkConsignmentSubmission",
+  fields: () => {
+    return {
+      displayText: {
+        type: GraphQLString,
+        resolve: (consignmentSubmission) => {
+          const statusEnum = {
+            draft: "Not displayed in My Collection",
+            submitted: "Submission in progress",
+            approved: "Submission in progress",
+            published: "Submission in progress",
+            rejected: "Submission in progress",
+            hold: "Submission in progress",
+            closed: "Submission evaluated",
+          }
+
+          return statusEnum[consignmentSubmission.state]
```

Comment:
What do you think about: 

```suggestion
          const statusDisplayTexts = {
            draft: "Not displayed in My Collection",
            submitted: "Submission in progress",
            approved: "Submission in progress",
            published: "Submission in progress",
            rejected: "Submission in progress",
            hold: "Submission in progress",
            closed: "Submission evaluated",
          }

          return statusDisplayTexts[consignmentSubmission.state]
```

Reply:
👍